templates/resources/datahub_aws_cluster.md.tmpl

```diff
@@ -9,11 +9,20 @@ description: |-
 
 {{ .Description | trimspace }}
 
-{{ if .HasExample -}}
-## Example Usage
+-{{ if .HasExample -}}
```

Comment:
Can you please remove this conditional as well. Now that we have renamed the example files, this evaluates to false, hence does not render the contents below. You can run `make docs` afterwards.

Reply:
removed